lib/resty/etcd/v3.lua

```diff
@@ -210,8 +303,9 @@ function refresh_jwt_token(self)
             password     = self.password,
         }
     }
-    local res, err = _request_uri(self, 'POST',
-                        choose_endpoint(self).full_prefix .. "/auth/authenticate",
+    local endpoint= choose_endpoint(self)
```

Comment:
seems we don't handle when the endpoint id nil ?

Reply:
got, the code is cover, and I will add test case for this